about.html

```diff
@@ -3,14 +3,143 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200;400;500;600&display=swap" rel="stylesheet">
   </head>
-  <body>
+  <body class="margin-t-l" >
     <nav class="nav-bar">
```

Comment:
The navbars have different html structure. You'll see that layout-container does not exist in index.html